docker/native/launch

```diff
@@ -42,11 +42,11 @@ result=$(/opt/kafka/kafka.Kafka setup \
       --default-configs-dir /etc/kafka/docker \
       --mounted-configs-dir /mnt/shared/config \
       --final-configs-dir /opt/kafka/config \
-      -Dlog4j.configuration=file:/opt/kafka/config/tools-log4j.properties 2>&1) || \
+      -Dlog4j2.configurationFile=file:/opt/kafka/config/tools-log4j2.yaml 2>&1) || \
```

Comment:
`/opt/kafka/config/tools-log4j2.yaml` is nonexistent in running the setup. Maybe we can just remove it to align the command to jvm dockerfile